server/routes/lists.js

```diff
@@ -101,7 +105,7 @@ const changeList = async (req, res) => {
       res.status(400).send({ errors: [{ msg: "No list with given ID." }] });
       return;
     }
-    res.status(201).send();
+    res.status(201).send({ message: "list updated successfully" });
```

Comment:
is this message purposeful when the status code exists?

Reply:
It's totally optional. just wanna abuse the snackbar RJ made as it's so handy to use. 🤖